### PR TITLE
Code Bridge: small console updates

### PR DESCRIPTION
**Pull request by molly-moen**
A couple updates to the code bridge console:
- On level change, reset the console. Previously the contents would stick around until you manually clicked the clear button, which is a bit confusing when all your code changes
- Change the clear console button from a broom to an eraser, to match current mocks.
- Remove the hard-coded `[PYTHON LAB]` prefix from the start of every system message, and instead make the prefix dependent on the lab app name. For now only python lab adds a system message prefix, as it's the only one that uses this console.

For the clear console logic, I did something similar to #58967 so that the console wouldn't be cleared on hot reload. This has an interesting side effect that is only present if you are in a level progression that switches from Python Lab to another lab2 lab (and therefore never reload the page). In this case the console contents persist, because the level never changed from the console's point of view. This seems fine to me as the console contents were for that level anyway, and this is probably a very rare situation to come across. 

## Links

- jira ticket: [CT-614](https://codedotorg.atlassian.net/browse/CT-614)

## Testing story
Tested locally.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/codebridge/Console/index.tsx [hunk 1]
```diff
@@ -11,6 +11,19 @@ import moduleStyles from './console.module.scss';
 const Console: React.FunctionComponent = () => {
   const codeOutput = useAppSelector(state => state.codebridgeConsole.output);
   const dispatch = useDispatch();
+  const levelId = useAppSelector(state => state.lab.levelProperties?.id);
+  const previousLevelId = useRef(levelId);
+  const appName = useAppSelector(state => state.lab.levelProperties?.appName);
+
+  const systemMessagePrefix = appName === 'pythonlab' ? '[PYTHON LAB] ' : '';
```
Comment (ebeastlake) [inline apps/src/codebridge/Console/index.tsx:19]: Should we add a TODO to reference updating this with other appNames eventually?

**Review by ebeastlake — APPROVED** [review]

File: apps/src/codebridge/Console/index.tsx (hunk 1 above)
Comment (molly-moen) [reply apps/src/codebridge/Console/index.tsx:19]: good call, will update

File: apps/src/codebridge/Console/index.tsx [hunk 2]
```diff
@@ -11,6 +11,20 @@ import moduleStyles from './console.module.scss';
 const Console: React.FunctionComponent = () => {
   const codeOutput = useAppSelector(state => state.codebridgeConsole.output);
   const dispatch = useDispatch();
+  const levelId = useAppSelector(state => state.lab.levelProperties?.id);
+  const previousLevelId = useRef(levelId);
+  const appName = useAppSelector(state => state.lab.levelProperties?.appName);
+
+  // TODO: Update this with other apps that use the console as needed.
+  const systemMessagePrefix = appName === 'pythonlab' ? '[PYTHON LAB] ' : '';
```
Comment (thomasoniii) [inline apps/src/codebridge/Console/index.tsx:19]: this is fine as is, but over the slightly longer term this should probably be broken out into a separate function `getSystemMessagePrefix` or something, so we have a more clear separation of concerns

**Review by thomasoniii — APPROVED** [review]
added a future refactoring note, but otherwise lgtm 🚀

File: apps/src/codebridge/Console/index.tsx (hunk 2 above)
Comment (molly-moen) [reply apps/src/codebridge/Console/index.tsx:19]: Yep makes sense, this was a quick measure to remove the hard coded python lab string.

**Pull request merged**